templates/settings-admin.php

```diff
@@ -53,6 +53,14 @@
 	<h2><?php p($l->t('Storage'));?></h2>
 	<p><?php p($l->t('Users:'));?> <em id="numUsersStorage"><?php p($_['storage']['num_users']);?></em></p>
 	<p><?php p($l->t('Files:'));?> <em id="numFilesStorage"><?php p($_['storage']['num_files']);?></em></p>
+	<p><?php p($l->t('Storages:'));?> <em id="numFilesStorages"><?php p($_['storage']['num_storages']);?></em></p>
+	<p><?php p($l->t('Free Space:'));?> <em id="numFilesFreeSpace"><?php p($_['system']['freespace']);?></em></p>
```

Comment:
`numFilesFreeSpace` is a weird name though,
I guess it should be `diskFreeSpace` because it's not related to the number of files

Reply:
I've decided to go with `systemDiskFreeSpace` as it also comes from the `SystemStatistics` class